domain/auth/auth.go

```diff
@@ -96,7 +97,9 @@ func CtxWithAccessToken(ctx context.Context, at AccessToken) context.Context {
 // read (GET) the resource at the /ping path. This is obviously
 // completely bogus right now, eventually need to look into something
 // for ACL/RBAC
-type Authorizer struct{}
```

Comment:
I think we should change the struct name to `CasbinRBACEnforcer` or `CasbinEnforcer`. I have been meaning to change the name of this for a while - I don't like that it's the same name as the interface that it implements, not very idiomatic.

You'll then need to change the struct comment to 

// CasbinEnforcer holds the casbin.Enforcer struct

and any other references below should change as well.